README.md

```diff
@@ -1,2 +1,26 @@
-# kodo
-##### testing
+**<h1>Command line utilities using cobra and Go</h1>**
+>A go program to develop command line utilities with commands, subcommands, aliase etc.
+
+**Dependencies**
+> "github.com/spf13/cobra" <br />
+> "k8s.io/client-go/kubernetes" <br />
+> "k8s.io/client-go/rest" <br />
+> "k8s.io/apimachinery/pkg/apis/meta/v1" <br />
+> "github.com/openshift/api/route/v1" <br />
+> "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1" <br />
+> "k8s.io/api/apps/v1" <br />
+> "k8s.io/api/core/v1" <br />
+> "k8s.io/apimachinery/pkg/apis/meta/v1" <br />
+
+**How to Install**
+> git clone https://github.com/cli-playground/kodo.git
```

Comment:
Code needs to be formatted.

Reply:
Yes, changed, and will be available in next PR.

Reply:
Push your changes to this branch/PR please.